c10/core/TensorImpl.h

```diff
@@ -408,6 +408,10 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
     return numel_;
   }
 
+  virtual int64_t element_size() const {
+    return static_cast<int64_t>(storage_.itemsize());
```

Comment:
element_size is already defined on `TensorBody.h`, why do we need to duplicate it?

Reply:
Removed.